libs/langgraph/src/prebuilt/react_agent_executor.ts

```diff
@@ -223,6 +238,19 @@ export type CreateReactAgentParams<
   /** An optional list of node names to interrupt after running. */
   interruptAfter?: N[] | All;
   store?: BaseStore;
+  /**
+   * An optional schema for the final agent output.
+   *
+   * If provided, output will be formatted to match the given schema and returned in the 'structured_response' state key.
+   * If not provided, `structured_response` will not be present in the output state.
+   *
+   * Can be passed in as:
+   *   - Zod schema
+   *   - Dictionary object
+   *   - [prompt, schema], where schema is one of the above.
+   *        The prompt will be used together with the model that is being used to generate the structured response.
+   */
+  responseFormat?: StructuredResponse | [string, StructuredResponse];
```

Comment:
I know we do this everywhere in Python but would really like to not use tuples ever again in JS